### ISSUE TITLE
Small tweaks for inbox and wallet switching

### ISSUE THREAD

**Issue by creyeknife** [O3Labs/O3Android#311]
Badge on inbox icon is erractic. Shows up when loading, even if there is no messages, and you have to go through each page it's shown to get rid of it. Then it will come back again next time you open the app. Should only show when new message received and hide on all pages when inbox is viewed.

Occasionally there is some heavy lag when swiping back to the total view. 5 to 10 seconds freezing.

Updating the name of a watch address on the wallet selector doesn't seem to work. 

Hide the headings of the groups that have no content in them. So if not hidden watch address, hide the heading. 

When loading the app, auto select the default wallet and the user can just enter their print to get in to the app.

CTA button on inbox messages are a little far from the content and feel disconnected. Reduce top padding by 50%. Maintain bottom margin as it is now. 

For hidden watch address, and maybe even all watch address, I still would like to see an option for view in explorer. where you can get more details like transactions and everything from explorer.o3

**Comment by saltyskip** [comment]
"When loading the app, auto select the default wallet and the user can just enter their print to get in to the app."

Let's put this in a seperate issue for reworking all of the passcode dialogs to soemthing custom. Right now the current default lib i use is not very flexible

**Comment by creyeknife** [comment]
Yeah cool, actually it would be nice to go customer and make the experience a bit more fluid with O3 if there is times.
